src/finance.py

```diff
@@ -102,7 +102,14 @@ def get_info(symbol: str) -> dict[str, str]:
         return_dict["currency"] = ticker.info["financialCurrency"]
         return_dict["sector"] = ticker.info["sector"]
     except:
-        data = yf.download(return_dict["ticker"], period="1d", progress=False)
+        range = "1d"
```

Comment:
This try-except is a bit difficult to follow. Consider adding a comment before this block.

We should avoid using a bare `except` too. It looks like it should handle a `KeyError`, so you should look to change this.